### PR TITLE
Auto wormholing only on the 100th levels

### PR DESCRIPTION
**Pull request by GreatOldOne**
Wormholes will be used only in the 100th levels (jumps 10 levels instead
of 1).

### CODE REVIEW

File: autoPlay.user.js [hunk 1]
```diff
@@ -1238,6 +1244,18 @@ function isRainingRound(level)
 	}
 }
 
+function isWormholeRound(level)
```
Comment (rossengeorgiev) [inline autoPlay.user.js]: :open_mouth: 
`return level % WORMHOLE_INFO.LEVEL === 0`

File: autoPlay.user.js [hunk 2]
```diff
@@ -1238,6 +1244,18 @@ function isRainingRound(level)
...
+{
+	var mod = level % WORMHOLE_INFO.LEVEL;
+
+	if (mod === 0) {
+		return true;
```
Comment (rossengeorgiev) [inline autoPlay.user.js:1248]: :open_mouth: 
`return level % WORMHOLE_INFO.LEVEL === 0`

Comment (GreatOldOne) [inline autoPlay.user.js:1248]: Samething, mine is just verbose to combine with the other functions that does the same kinda of stuff.
Semantics...

**Comment by GreatOldOne** [comment]
Rebased and ready to be merged. This needs high priority.

**Comment by GreatOldOne** [comment]
I can. Let me submit a new patch.

**Comment by GreatOldOne** [comment]
Rebased!

**Comment by GreatOldOne** [comment]
Ops... Forgot to change the function aswell. sec

**Comment by GreatOldOne** [comment]
Everything should be fine now.

**Comment by GreatOldOne** [comment]
Let me know if you need me to change anything. Otherwise merge as soon as possible for everyone to be ready for tomorrow's end game.

**Comment by xPaw** [comment]
It looks fine, but I have a feeling many wormholes will get unused because we won't hit many levels that are dividable by 100.

**Comment by GreatOldOne** [comment]
Merge this one, and I'll start working on the startMinutes asap.

**Comment by GreatOldOne** [comment]
I did some improvements to start the end game nuking before an hour.
May I push this patch here or do you merge it first and I send a new PR?

**Pull request merged**